Sources/WhoopDIKit/Container/ServiceDictionaryTaskLocal.swift

```diff
@@ -0,0 +1,26 @@
+enum ServiceDictionaryTaskLocal {
+    @TaskLocal
+    static var dictionary = ServiceDictionaryTaskLocalWrapper()
```

Comment:
I think this being static would cause multiple containers to share the same local dependency graph, is that correct?

Reply:
since this is `TaskLocal` it only sticks around for the given task. They all have the same base wrapper (which is an empty container), but when you go into the task (using the `ServiceDictionaryTaskLocal.$dictionary.withValue`), it will update the value to be the new one. Here are the docs: https://developer.apple.com/documentation/swift/tasklocal

Reply:
For tasks - sure, but if you are just on the main thread and have two containers I think this could lead some funkiness. 

This is kind of convoluted use case but something like this would lead to some weird behavior I think:

```swift
let containerA = Container()
let containerB = Container()

// Add modules to containers

containerA.inject { module in
     module.factory { 
          let depB: DependencyB = containerB.inject { ... }
          return depB
     }
}
```

Is there anyway to use TaskLocal via an instance variable so it can be scoped to a container? (Also please let me know if my understanding above is incorrect).